lib/BootBot.js

```diff
@@ -292,16 +292,20 @@ class BootBot extends EventEmitter {
 
     this._hearMap.forEach(hear => {
       if (typeof hear.keyword === 'string' && hear.keyword.toLowerCase() === text.toLowerCase()) {
-        captured = true;
-        return hear.callback.apply(this, [event, new Chat(this, senderId), {
-          keyword: hear.keyword
+        let res = hear.callback.apply(this, [event, new Chat(this, senderId), {
+          keyword: hear.keyword,
+          captured: captured
```

Comment:
Use shorthand for implicit value here (simply `captured` without the `: captured` part).